<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterDialog</class>
 <widget class="QDialog" name="FilterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout_4">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Color</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>VehicleModel</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>VehicleDesc</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>ModelYear</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Type</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>DateStolen</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Location</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="colorEdit"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="modelEdit"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="descEdit"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="yearEdit"/>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="typeEdit"/>
       </item>
       <item row="5" column="1">
        <widget class="QCalendarWidget" name="calender"/>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="locationEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>SAVE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="text">
        <string>DELETE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>CANCEL</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QTableView" name="filterView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
